regolith/schemas.py

```diff
@@ -3620,6 +3621,172 @@
+                          "required": False,
+                          "type": "list"},
+                "status": {"description": "current state of deliverable",
+                           "required": False,
+                           "type": "string",
+                           "eallowed": PROJECTA_STATUS},
+            }
+        },
+        "description": {
+            "description": "explanation of projecta",
+            "required": False,
+            "type": "string"
+        },
+        "end_date": {
+            "description": "projecta end date, yyyy-mm-dd. "
+                           "Not necessarily the same as deliverable due date",
+            "required": False,
+            "anyof_type": ["date", "string"]
+        },
+        "grants":{
+            "description": "grant id funding the project",
+            "required": False,
+            "type": "string"
+        },
+        "group_members": {
+            "description": "group member id's working on this project",
+            "required": False,
+            "type": "list"
+        },
+        "kickoff":{
+            "description": "details the projecta kickoff meeting",
+            "required": False,
+            "type": "dict",
+            "schema": {
+                "date": {"description": "kickoff meeting date, yyyy-mm-dd.",
+                         "required": False,
+                         "anyof_type": ["date", "string"]},
+                "due_date": {"description": "kickoff meeting by date, yyyy-mm-dd.",
+                             "required": False,
+                             "anyof_type": ["date", "string"]},
+                "name": {"description": "name of meeting",
+                         "required": False,
+                         "type": "string"},
+                "objective": {"description": "goal of the meeting",
+                              "required": False,
+                              "type": "string"},
+                "audience": {"description": "rolls of audience members "
+                                            "e.g. 'pi', 'lead'",
+                             "required": False,
+                             "type": "list"},
+                "notes": {"description": "any notes about the kickoof",
+                          "required": False,
+                          "type": "list"},
+                "status": {"description": "proposed, started, finished, converged",
+                           "required": False,
+                           "type": "string",
+                           "eallowed": PROJECTA_STATUS}
+
+            }
+        },
+        "lead": {
+            "description": "id of projecta lead",
+            "required": False,
+            "type": "string"
+        },
+        "log_url": {
+            "description": "link to log of meetings",
+            "required": False,
+            "type": "string"
+        },
+        "milestones": {
```

Comment:
the kickoff and deliverables are also kind of milestones, so please harvest text from their descriptions and synchronize (i.e., if you change it based on my comments there, make sure the changes appear here).

I probably made a mistake to have the kickoff field.  It is just a milestone with the same structure.  But I am not ready to refactor all the code to change that yet.